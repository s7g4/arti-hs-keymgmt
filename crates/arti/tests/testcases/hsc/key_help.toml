bin.name = "arti"
args = "hsc key --help"
status.code = 0
