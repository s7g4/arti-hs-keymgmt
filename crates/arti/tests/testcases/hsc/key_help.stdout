Key management subcommands

Usage: arti hsc key [OPTIONS] <COMMAND>

Commands:
  get     Get or generate a hidden service client key Deprecated. Use key get instead
  rotate  Rotate a hidden service client key
  remove  Remove a hidden service client key
  help    Print this message or the help of the given subcommand(s)

Options:
  -c, --config <FILE>                 Specify which config file(s) to read. Defaults to [File("[..]"), Dir("[..]")]
  -o <KEY=VALUE>                      Override config file parameters, using TOML-like syntax.
  -l, --log-level <LEVEL>             Override the log level (usually one of 'trace', 'debug', 'info', 'warn', 'error').
      --disable-fs-permission-checks  Don't check permissions on the files we use.
  -h, --help                          Print help
